Update README.md on https://halcyonsim.org

life777eternal opened the pull request:
Added something to the page. Was blank before, with just a title.  How can we expect more developers to get interested without anything on the website?

life777eternal committed:
Update README.md

Added something to the page. Was blank before, with just a title.

life777eternal committed:
Update README.md

life777eternal committed:
Update README.md

life777eternal committed:
Update README.md

Some people don't know what HalcyonGrid is, nor what OpenSim is, that's why it must be explained to them.
Thank you.

kf6kjg commented on README.md:
```diff
@@ -1 +1,26 @@
+
+Simulates a virtual world environment. With sun, moon, clouds, land, buildings *(objects with seamless textures to give the desired look)*, avatars.. and interactive objects that are scripted.
+
+HalcyonGrid or HalcyonSim was a fork from the <a href="http://opensimulator.org" target="_blank" rel="nofollow" >OpenSimulator</a> Project in 2010, 
```
I'd prefer to use the normal markdown link format. When I started web development I used to use `target="_blank"` a lot, but since then I've realized that as a user it's my choice to open a new tab or not, not the author of the page's choice how I navigate.

life777eternal replied:
The normal markdown link format works, aye, though the `rel` tag can't be used on the markdown link format, and have removed the `target="_blank"` in the anchors, that's a reasonable reason to not have them in there. Although on regular websites not through github, I think the `target="_blank"` is useful for links on external websites.

life777eternal replied:
> Additionally I'm not sure that this will help the website as I don't think the readme feeds the site. The website code is expected to be created using a different set of tools I haven't completed setting up.

The website [https://halcyonsim.org](https://halcyonsim.org) has at the "bottom"  "This site is open source. [Improve this page](https://github.com/HalcyonSim/halcyonsim/edit/master/README.md)." That link on the words "Improve this page" points to this repository. (which is pretty much at the top right now, since there's nothing on the page, but the title and that phrase.)

kf6kjg commented on README.md:
```diff
@@ -1 +1,26 @@
\ No newline at end of file
+# HalcyonSim
+
+Simulates a virtual world environment. With sun, moon, clouds, land, buildings *(objects with seamless textures to give the desired look)*, avatars.. and interactive objects that are scripted.
```
Fragmented sentence. Please edit.
Was that `..` supposed to be a comma?

life777eternal replied:
Please re-read the updated version and let me know if there's still fragmented sentence(s) still and have removed the double dots (`..`) in the new version. Thank you.

kf6kjg commented on README.md:
```diff
@@ -1 +1,26 @@
+
+## Virtual World where people may move about in a full 3D world.
+
+Halcyon World is a Virtual World with a full 3D environment,  written in C Sharp (C#), where you may move about with a character known as an avatar in a 3D World accessed with a 3D Graphics Application, commonly known as a "Viewer," which is used to access these virtual worlds. (not a VR headset) such as the most common one, <a href="https://firestormviewer.org" target="_blank" rel="follow">Fires ...
```
Double spaces should be single spaces.
"Graphics Application" is not a title and thus should not be capitalized.
Commas go outside of quotes, which are also unneeded in this case.
Several run-on sentences here.

life777eternal replied:
The double spaces are removed in the updated version, and have removed the capitals on "Graphics Application". Which quotes did you feel are unneeded?  Incorrect grammar usually sticks out to me like a sore thumb, though I was trying to merge two different versions of the same thing, and I might have been tired at the time. Let me know if you still see run-on sentences in the new version. Thank you.

kf6kjg replied:
The quotes I mentioned are those that are being used like the commonly misused air quotes in speech. Some examples:
- `commonly known as a "Viewer,"` could simply be `commonly known as a viewer,` without any semantic loss.
- `It allows users "talk" to` likewise could be `It allows users to talk to` - with the additional detail of fixing the grammar with the additional `to`. If you are concerned that the word "talk" contains too much semantic meaning of verbal communication, then simply choose another word that doesn't convey that meaning. For me the word is fine, it just doesn't need the quotes.

I hope that from those few examples you can extrapolate the kind of change requested to the rest of the prose.

Additionally I see `IM's` - that's a possessive form being used as a plural.

kf6kjg reviewed (changes requested):
Thank your for your effort. However writing good English is hard. My comments only address some of what I ran across in this writeup.

Additionally I'm not sure that this will help the website as I don't think the readme feeds the site. The website code is expected to be created using a different set of tools I haven't completed setting up.

life777eternal committed:
Update README.md

Thanks kf6kjg for the suggestions. Have removed the target="_blank" in the links, though kept the regular anchor style since the rel tag can't be used with the other link style. Let me know how it looks.  Thank you.

life777eternal committed:
Update README.md

Removed comma inside quote, removed the capitals on "Graphics Application", separated a run-on sentence. Removed extra space between a couple sentences.

life777eternal committed:
Update README.md

Made a few more wording changes, and added the hyperlink for Mundos Market.

life777eternal committed:
Update README.md

Added FSFE.org link to "there is no "cloud" only somebody else's computers" and added Archive.org link for the last saved version of InWorldz.com on Aug. 6, 2018 before it was ended. (the domain was bought in 2019)  Thank you.

life777eternal committed:
Update README.md

Not sure why you prefer to have the main heading title at the top in all lowercase, though I guess it wouldn't hurt. We can try it like that for a while.  Thank you.

life777eternal committed:
New expanded Update of README.md

Added a couple more titles, and split up a paragraph, and added some more information, and links. Changed the "registered" logo beside "Second Life" to the TradeMark logo "™" as the Second Life website now seems to have that one.

Thank you, Shalom.

kf6kjg commented on README.md:
```diff
@@ -1,14 +1,20 @@
 ## Virtual World where people may move about in a full 3D world.
 
-Halcyon World is a Virtual World with a full 3D environment,  written in C Sharp (C#), where you may move about with a character known as an avatar in a 3D World accessed with a 3D graphics application, commonly known as a "Viewer" which is used to access these virtual worlds. (not a VR headset) such as the most common one, <a href="https://firestormviewer.org" rel="follow">FirestormOS</a>.  *(th ...
+Halcyon World is a Virtual World with a full 3D environment,  written in C Sharp (C#), where you may move about with a character known as an avatar in a 3D World accessed with a 3D graphics application, commonly known as a "viewer", which is used to access/login to these virtual worlds (not a VR headset). The most common one currently is the <a href="https://firestormviewer.org" rel="follow">Fire ...
```
Please omit the jab `(which most OpenSim Grids do not have)`.

Please omit `Although not yet with <a href="https://OpenPGP.org" rel="nofollow">OpenPGP</a> Encryption.` If such a think should exist, it should be in an issue ticket not here in the README - and even then that's more a client thing than a server thing.

Either rework to be part of the flow or omit `(Or even exported from a Grid as Mesh/DEA.)` Parenthetical insertions are not a good writing style, and wouldn't be proper grammar wrapping a partial sentence like this anyway.

life777eternal replied:
Alright, thank you, I'm making those changes... and OpenPGP is actually both a client and server thing, at least when sending mail directly from a server. Users/Clients need a [OpenPGP](https://OpenPGP.org) compatible software, and the server does as well, to encrypt the message with each user's public key. Then the user/client/recipient decrypts the message with their private key.

kf6kjg commented on README.md:
```diff
@@ -1,14 +1,20 @@
-Halcyon is different and incompatible with both Second Life© and OpenSims, although we can use the OpenSim viewers (<a href="https://firestormviewer.org/os-operating-system/" rel="follow" >FirestormOS</a> recommended). Halcyon has better physics & script engines, more prims, free upload (depending on each world's setting), and a more powerful asset database (NOT a "cloud" ... <a href="https://fsf ...
+# Different 
+
+Halcyon is different and incompatible with both Second Life™ and OpenSims, although we can use the OpenSim viewers (<a href="https://firestormviewer.org/os-operating-system/" rel="follow" >FirestormOS</a> recommended, and currently <a href="https://discover.creomundos.com/ViewerDownloads.aspx" rel="follow">6.6.14</a> is the last compatible with HalcyonGrid). Halcyon has better physics & script en ...
```
I think this entire paragraph needs a rethink from the ground up. There's no such thing as `OpenSims`, there is "OpenSim" and there are other forks thereof.I recommend moving the FirestormOS recommendation out of the parenthetical block. In fact I'm not quite sold that this paragraph nor this readme is the best place to be recommending viewers anyway.

Jabs and snide comments don't belong in README files.

kf6kjg commented on README.md:
```diff
@@ -1,14 +1,20 @@
+
+# For anyone to setup and run
+
+Halcyon is intended for anybody to run their own world, and there already are several, both public and private. (Even with a project of inter-worlds communication.) Although setting up a Halcyon World is not without it's complexities, and it currently only runs properly with Physics on Windows Servers. Although this needs a restart of the Halcyon development, which has been very slow since the en ...
```
Unneeded parenthesis, incorrect use of `it's`. Probably too early yet to mention Mundos Marketplace. Wait until the release and then we can think about it.

life777eternal committed:
Updated README.md for HalcyonSim.org website

Removed the part about OpenPGP Encryption and the part before that. Commented out the part about the Mundos Market, for later uncommenting out.

Thank you, Shalom.

kf6kjg commented on README.md:
```diff
@@ -1 +1,23 @@
+
+# Different Type of Grid
+
+Halcyon is different and incompatible with both Second Life™ and OpenSim Grids, although we can use the OpenSim viewers. Halcyon has better physics & script engines, more prims, free upload (depending on each individual world's setting), and a more powerful asset database (Not a "Cloud" ... <a href="https://fsfe.org/contribute/spreadtheword.html#nocloud" rel="follow" >*there is no "cloud" just ot ...
```
I just checked that FSFE link... Why are you attempting to place marketing materials here?

life777eternal replied:
Well I guess you could consider that as "marketing materials", though it's true, there is no "cloud"... just other people's computers. And that's the **Free Software Foundation Europe** website. Hence the "FSFE.org". But you're probably right that that's not necessary to be there, and now that you mention it, that may not actually be true for every Halcyon World, that they're not using the "cloud" computing. Sorry, I have removed it in a newer version.

Thank you, Shalom.

kf6kjg commented on README.md:
```diff
@@ -1 +1,23 @@
+
+## Virtual World where people may walk in a full 3D world.
+
+Halcyon World is a Virtual World with a full 3D environment,  written in C Sharp (C#), where you may move about with a character known as an avatar in a 3D World accessed with a 3D graphics application, commonly known as a "viewer", which is used to access/login to these virtual worlds (not a VR headset). The most common one currently is the <a href="https://firestormviewer.org" rel="follow">Fire ...
```
FYI Blender actually has its own website: https://www.blender.org/

kf6kjg replied:
Not that we actually need to link to any 3D model editor in the first place...

life777eternal replied:
Ok, I see.. well I don't think there would be any problem linking to a website for an Open Source program.

life777eternal committed:
Update README.md

Changed the linked website for Blender to their own actual website as noted. Added "Open Source".

Thank you, Shalom.

life777eternal committed:
Updated README.md for HalcyonSim.org website

Changed two heading styles from # to ##.

life777eternal committed:
Updates README.md for the HalcyonSim.org website.

Removed the Free Software Foundation web link and the rest with it.

Thank  you, Shalom.

life777eternal committed:
Updated README.md for halcyonsim.org website

Several more additions, please note changes, and let me know if  you have any questions.

Thank you, Shalom.

life777eternal committed:
Update README.md

Made a few additions/updates, and added some hyperlinks. 

Thank you, Shalom.

life777eternal committed:
Update README.md

Removed the quotes on viewer.
